lib/ansible/modules/files/find.py

```diff
@@ -407,7 +408,10 @@ def main():
     looked = 0
     for npath in params['paths']:
         npath = os.path.expanduser(os.path.expandvars(npath))
-        if os.path.isdir(npath):
+        try:
+            if not os.path.isdir(npath):
+                raise Exception("'%s' is not a directory" % to_text(npath))
```

Comment:
Shouldn't exceptions be using `to_native` not `to_text`.

Reply:
in core code, in modules, they get jsonified so to_text ends up being correct

Reply:
But this is an exception, I assume there is some internal logic where getting the text of an exception that has been raised/caught requiring the use of to_native.

Reply:
good point, its not a 'final raise', i'll change